duckduckgo/iOS
# swifty clean up

remove references to swifty json hanging around in project folders and remove seemingly unused file with references to swifty and previous contractors

Task/Issue URL: Dev / clean up tasks
Tech Design URL:
CC:

**Description**:

Performing a clean checkout and running run_tests.sh fails at the moment because of references to swifty json (deleting Carthage/ folder and running tun_tests.sh does the same)

**Steps to test this PR**:
1. Perform a clean checkout of this branch and run run_tests.sh - it should pass
1. Launch the app and confirm it hangs together


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
